docs/userguides/siemexample.md

```diff
@@ -1,57 +1,63 @@
-# Ingest file event data into a SIEM tool
+# Ingest file event data or alerts into a SIEM tool
 
-This guide provides instructions on using the CLI to ingest Code42 file event data
+This guide provides instructions on using the CLI to ingest Code42 file event data or alerts 
 into a security information and event management (SIEM) tool like LogRhythm, Sumo Logic, or IBM QRadar.
 
 ## Considerations
 
-To ingest file events into a SIEM tool using the Code42 command-line interface, the Code42 user account running the integration
+To ingest file events or alerts into a SIEM tool using the Code42 command-line interface, the Code42 user account running the integration
 must be assigned roles that provide the necessary permissions.
 
 ## Before you begin
 
 First install and configure the Code42 CLI following the instructions in
 [Getting Started](gettingstarted.md).
 
-## Run file event queries
-You can get security events in either a JSON or CEF format for use by your SIEM tool. You can query the data as a
+## Run queries
+You can get file events in either a JSON or CEF format for use by your SIEM tool. Alerts data is available in either JSON or RAW-JSON format. You can query the data as a
```

Comment:
Technically file events can be raw as well

Reply:
The phrasing makes it sound like RAW-JSON is only an Alerts thing

Reply:
Good call, I updated it to remove RAW-JSON.